### DISCUSSION TITLE
Using Hero with overlay image is also affecting text

### DISCUSSION THREAD

**Discussion by ayan-akd** [saadeghi/daisyui#2177] [Q&A]
When I'm using Hero with overlay image, it is also affecting the opacity of text. Any way to fix that?

**Comment by saadeghi** [comment]
Please share a code example

**Reply by ayan-akd** [reply]
Just check this one https://daisyui.com/components/hero/
It's the last one called : Hero with overlay image

**Reply by saadeghi** [reply]
I don't see any text opacity there.
